shared/constants/types/fs.js

```diff
@@ -79,3 +95,76 @@ export const stringToPathType = (s: string): PathType => {
 export const pathTypeToString = (p: PathType): string => p
 export const pathConcat = (p: Path, s: string): Path =>
   p === '/' ? stringToPath('/' + s) : stringToPath(pathToString(p) + '/' + s)
+export const pathIsNonTeamTLFList = (p: Path): boolean =>
+  pathToString(p) === '/keybase/private' || pathToString(p) === '/keybase/public'
+
+export type sortSettingDisplayParams = {
+  sortSettingText: string,
+  sortSettingIconType: IconType,
+}
+
+export const sortSettingToIconTypeAndText = (s: SortSetting): sortSettingDisplayParams => {
```

Comment:
if this is only used by this one component it doesn't need to be here, just put it into the component